package/boot/uboot-envtools/files/ath79

```diff
@@ -18,7 +18,7 @@ buffalo,wzr-hp-ag300h)
         ;;
 buffalo,bhr-4grv2|\
 engenius,ecb1750|\
-glinet,gl-ar300m-nor|\
+glinet,gl-ar300m-*|\
```

Comment:
Hi, I personally dislike using wildcards for this task as they are determined to be overlooked when adding/changing devices (just recently had a case like this).
Since this is only affecting three devices, I would strongly suggest just adding those three explicitly.